pom.xml

```diff
@@ -280,6 +281,12 @@
 
   <profiles>
     <profile>
+      <id>spark_core-2.10</id>
+      <properties>
+        <spark.core.artifactid>spark_core_2.10</spark.core.artifactid>
+      </properties>
+    </profile>
+    <profile>
```

Comment:
Maven can run with multiple profiles. This profile only changes the spark core version and uses default for others.